src/WeekComponent/WeekComponent.css

```diff
@@ -6,76 +6,99 @@
 .description-week-day {
   height: 26px;
   align-self: center;
-  padding: 10px;
+  padding: 15px 178px 24px 0px;
```

Comment:
No need for exact paddings when definding elements, since it's ok to have variations in rendering between devices